receiver/sqlserverreceiver/scraper.go

```diff
@@ -300,3 +329,231 @@ func (s *sqlServerScraperHelper) recordDatabaseStatusMetrics(ctx context.Context
 
 	return errors.Join(errs...)
 }
+
+func (s *sqlServerScraperHelper) recordDatabaseQueryTextAndPlan(ctx context.Context, topQueryCount uint) (plog.Logs, error) {
+	// Constants are the column names of the database status
+	const DBPrefix = "db."
```

Comment:
```suggestion
	const dbPrefix = "db."
```
to avoid giving the impression we're exporting this constant.

Reply:
fixed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37958/commits/9d926cd3bae0eb2c361c72a9b3f4190f8ee7a1b7